specs/agents/metadata.md

```diff
@@ -37,10 +37,13 @@ if os == windows
   hostname = exec "cmd /c hostname"                   // or any equivalent *
   if (hostname == null || hostname.length == 0)
     hostname = env.get("COMPUTERNAME")
+  domain = env.get("USERDNSDOMAIN")
+  if (domain != null && domain.length >0)
+    hostname += "." + domain
 else 
   hostname = exec "uname -n"                          // or any equivalent *
   if (hostname == null || hostname.length == 0)
-    hostname = exec "hostname"                        // or any equivalent *
+    hostname = exec "hostname --fqdn"                 // or any equivalent *
```

Comment:
```suggestion
    hostname = exec "hostname --f"                 // or any equivalent *
```

The `--fqdn` switch name is Linux-only, I think. On macOS, at least, it only supports `hostname -f`.

Reply:
good point, let's keep the short version for more compatibility.